index.bs

```diff
@@ -315,10 +332,9 @@ To <dfn>create a new animator instance</dfn> given a |name|, |timeline|, |effect
 
     5. Let |state| be <a>StructuredDeserialize</a>(|serializedState|).
 
-    6. Let |animatorInstance| be the result of <a>Construct</a>(|animatorCtor|, [|options|, |state|]).
-
-        If <a>Construct</a> throws an exception, abort the following steps.
-
+    6. Let |animatorInstance| be the result of <a>constructing</a> |animatorCtor| with
+      [|options|, |state| as args. Rethrown any exception from construction and abort the following
```

Comment:
Ditto; the abort needs to be conditional.

Reply:
Done.